TLM/TLM/Resources/Translations/Options.csv

```diff
@@ -98,18 +98,22 @@
-"General.Dropdown.Option:Low","Niedrig","Low","Bajo","Basse","Alacsony","Basso","低い","낮음","Laag","Niska","Baixa","Замедленное","低","低","Low","","Низька"
-"General.Dropdown.Option:Medium","Mittel","Medium","Medio","Moyenne","Közepes","Medio","中くらい","중간","Middelhoog","Średnia","Média","Среднее","高","中","Medium","","Середня"
-"General.Dropdown.Option:High","Hoch","High","Alto","Haute","Magas","Alto","高い","높음","Hoog","Wysoka","Alta","Высокое","高","高","High","","Висока"
-"General.Dropdown.Option:Very high","Sehr hoch","Very high","Muy alto","Très haute","Nagyon magas","Molto alto","とても高い","매우 높음","Zeer hoog","Bardzo wysoka","Muito Alta","Очень высоко","非常高","非常高","Very high","","Дуже висока"
+"General.Slider:GUI scale","Größe der Benutzeroberfläche","User interface scale","","Taille de l'interface","Grafikus felhasználói felület mérete","scala GUI","","GUI 스케일","Gebruikersinterface schaal","Skala interfejsu użytkownika","Escala da interface de usuário","Масштаб пользовательского интерфейса","","","User interface scale","Arayüz ölçeği","Масштаб графічного інтерфейсу користувача"
+"General.Tooltip.Format:GUI scale: {0}%","GUI-Skala: {0}%","Current GUI scale {0}%","","Info-bulle. Format : Échelle GUI : {0}%","Grafikus felhasználói felület {0}%","scala GUI: {0}%","","현재 GUI 스케일 {0}%","Huidige GUI-schaal {0}%","Aktualna skala interfejsu {0}%","Escala da Interface {0}%","Текущий масштаб интерфейса {0}%","","","Current GUI scale {0}%","Arayüz ölçeği: {0}%","Зараз встановлено масштаб {0}%"
+"General.Dropdown:Simulation accuracy","Genauigkeit der Simulation","Simulation accuracy","Precisión de la simulación","Précision de la simulation","Szimulációs pontosság","Precisione simulazione","シミュレーションの正確さ","시뮬레이션 정확도","Genauigkeit der Simulation","Dokładność symulacji","A precisão da simulação","Точность моделирования","模擬精準度","模拟真实度","Simulation accuracy","Simülasyon doğruluğu","Точність симуляції"
+"General.Dropdown.Option:Very low","Sehr niedrig","Very low","Muy bajo","Très basse","Nagyon alacsony","Molto basso","とても低い","매우 낮음","Zeer laag","Bardzo niska","Muito Baixa","Очень медленно","非常低","非常低","Very low","En düşük","Дуже низька"
+"General.Dropdown.Option:Low","Niedrig","Low","Bajo","Basse","Alacsony","Basso","低い","낮음","Laag","Niska","Baixa","Замедленное","低","低","Low","Düşük","Низька"
+"General.Dropdown.Option:Medium","Mittel","Medium","Medio","Moyenne","Közepes","Medio","中くらい","중간","Middelhoog","Średnia","Média","Среднее","高","中","Medium","Orta","Середня"
+"General.Dropdown.Option:High","Hoch","High","Alto","Haute","Magas","Alto","高い","높음","Hoog","Wysoka","Alta","Высокое","高","高","High","Yüksek","Висока"
+"General.Dropdown.Option:Very high","Sehr hoch","Very high","Muy alto","Très haute","Nagyon magas","Molto alto","とても高い","매우 높음","Zeer hoog","Bardzo wysoka","Muito Alta","Очень высоко","非常高","非常高","Very high","En yüksek","Дуже висока"
+"Shortcut.Modifier:Shift","","Shift","","","","","","","","","","","","","Shift","",""
```

Comment:
Out of curiosity why is it necessary to translate modifier keys?

Reply:
They are only translated for game-supported languages using the game code, and only while they are a part of a `InputKey` or `SavedInputKey`. We have more languages than the game. So at least modifier keys should be translated.